KsApi/models/User.swift

```diff
@@ -11,6 +11,7 @@ public struct User {
   public var location: Location?
   public var name: String
   public var needsFreshFacebookToken: Bool?
+  public var needsPassword: Bool?
```

Comment:
So if you check the schema for the GET `v1/users/self` in `https://staging.kickstarter.com/admin/api-docs/index.html` you'll see `needs_password` is not an optional, so can remove optionality for this property in this file.